### PR TITLE
Added rapidjson as a dependency

### PR DESCRIPTION
**Pull request by AWoloszyn**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -40,3 +40,6 @@
 	branch = release
```
Comment (devpow112) [inline .gitmodules:40]: Naming and patching can just use forward slashes like all the other submodules

File: .gitmodules [hunk 2]
```diff
@@ -40,3 +40,6 @@
...
 	ignore = dirty
 	shallow = true
+[submodule ".\\Externals\\rapidjson"]
+	path = .\\Externals\\rapidjson
```
Comment (devpow112) [inline .gitmodules:44]: Naming and patching can just use forward slashes like all the other submodules

Comment (devpow112) [reply .gitmodules:44]: Also add the `branch`, `ignore` and `shallow` properties please

Comment (AWoloszyn) [reply .gitmodules:44]: Sorry that was just what was automatic with git. Will FIx

**Review by devpow112 — DISMISSED** [review]

**Review by devpow112 — APPROVED** [review]

**Review by devpow112 — APPROVED** [review]

**Pull request merged**